x-pack/plugins/actions/server/lib/action_executor.ts

```diff
@@ -487,6 +488,9 @@ export class ActionExecutor {
         rawAction: rawAction.attributes,
       };
     } catch (e) {
+      if (isBoom(e, 404)) {
```

Comment:
Do the errors always be boom errors?

Reply:
Yes SavedObjectsClient throws Boom error for 404s

Reply:
We should use SavedObjectsErrorHelpers.isNotFoundError(e) provided by `@kbn/core/server`